client/blocks/login/login-form.jsx

```diff
@@ -109,20 +113,41 @@ export class LoginForm extends Component {
 	onSubmitForm = event => {
 		event.preventDefault();
 
-		const { usernameOrEmail, password } = this.state;
-		const { onSuccess, redirectTo } = this.props;
+		const { usernameOrEmail } = this.state;
+		const { accountType } = this.props;
+
+		if ( isRegularAccount( accountType ) ) {
```

Comment:
Could we check first if accountType is loaded, then split those 2 actions into 2 functions?
```js
onSubmitForm = event => {
    if ( ! this.props.accountTypeLoaded ) {
        return this.loadAccountType();
    }

    this.tryLogin();
};
```

Reply:
Hell yes, that's a very good idea!